src/Obj.hs

```diff
@@ -358,7 +358,8 @@ prettyUpTo max xobj =
             Instantiate _ -> ""
             External Nothing -> ""
             External (Just override) -> ")"
-            ExternalType -> ""
+            ExternalType Nothing -> ")"
```

Comment:
You can just match on `_` here.

Reply:
(to condense the two cases into one)

Reply:
The two cases doesn't return the same value. Is it fine if both returns `")"` ?

Reply:
Oh actually I made a mistake. I wanted to make that part just like `External` so  `ExternalType Nothing` should return "".

Reply:
Lovely!